### DISCUSSION TITLE
v0.5.0

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#15] [Announcements]
## What's Changed
* Fetching Homebrew Version with Github API by @stevenlei in https://github.com/teaxyz/teaBASE/pull/13
* Improve Emergency Kit PDF readability by @stevenlei in https://github.com/teaxyz/teaBASE/pull/11

**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.4.1...v0.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/teaBASE/releases/tag/v0.5.0'>v0.5.0</a>.</em>

**Comment by Ayaan1982** [comment]
Right sir 🙏
